res/skins/LateNight/skin.xml

```diff
@@ -46,12 +44,13 @@
       <attribute config_key="[Microphone],show_microphone">0</attribute>
       <attribute config_key="[Vinylcontrol],show_vinylcontrol">0</attribute>
       <attribute config_key="[PreviewDeck],show_previewdeck">0</attribute>
-      <attribute config_key="[Master],mixer_4decks_status">1</attribute>
       <attribute config_key="[Master],show_mixer">1</attribute>
       <attribute config_key="[Master],show_mics">0</attribute>
       <attribute config_key="[Library],show_coverart">1</attribute>
       <attribute config_key="[Library],show_library">1</attribute>
       <attribute config_key="[Library],show_current">1</attribute>
+      <attribute config_key="[Master],show_4decks">0</attribute>
```

Comment:
> You cannot set a start up value for the skin at on hand and state that it should persist.

Why not?  A skin may declare its on CO, like "my_ui_setting", and declare a default and declare that the value should persist.  That way when the skin first starts, it will have a correct default for my_ui_setting and if the user changes it, the value will be persisted.

But I agree that declaring it up here and then making it persist down there is poor -- all persist settings should be in the attributes section if possible.